src/test/java/library/util/file/CsvFileHandlerTest.java

```diff
@@ -0,0 +1,48 @@
+package library.util.file;
+
+import library.domain.Book;
+import library.domain.BookStatus;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+class CsvFileHandlerTest {
+
+    static private final String testFilePath = "src/main/resources/books.csv";
```

Comment:
```suggestion
    private static final String testFilePath = "src/main/resources/books.csv";
```

자바 컨벤션에서는 접근제한자가 가장 우선이지 않을까 싶습니다.

[Google Java Style Guide - Modifier Order](https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers)

Reply:
[fd1d79c](https://github.com/prgrms-be-devcourse/java-library-management/pull/21/commits/fd1d79c9c62b257851dc6ac629eb1be7c3bf2b8f)
반영했습니다!